### PR TITLE
Add app:start-services command

### PR DESCRIPTION
**Pull request by Aschen**
## What does this PR do?

Adds the `--services-only` flag to the `instance:spawn` command to run only Kuzzle dependencies (ES & Redis)

Should be `kourou app:start-services`

### How should this be manually tested?

```
kourou instance:spawn --services-only
```

### CODE REVIEW

**Review by Njuelle — APPROVED** [review]

**Review by scottinet — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -257,6 +259,38 @@ OPTIONS
 
 _See code: [src/commands/api-key/search.ts](src/commands/api-key/search.ts)_
 
+## `kourou app:scaffold NAME`
+
+Scaffold a new Kuzzle application
```
Comment (scottinet) [inline README.md:264]:
```suggestion
Scaffolds a new Kuzzle application
```

File: README.md [hunk 2]
```diff
@@ -257,6 +259,38 @@ OPTIONS
...
+## `kourou app:start-services`
+
+Starts Kuzzle services (Elasticsearch and Redis)
+
+```
+USAGE
+  $ kourou app:start-services
+
+OPTIONS
+  --check  Check prerequisite before running services
```
Comment (scottinet) [inline README.md:288]:
```suggestion
  --check  Check prerequisites before running services
```

**Review by scottinet — CHANGES_REQUESTED** [review]
why does this PR now add an `app:scaffold` command?

**Review by rolljee — APPROVED** [review]

**Comment by Aschen** [comment]
> why does this PR now add an `app:scaffold` command?

It's only the documentation, I forgot to generate the documentation in the previous PR :-1:

**Review by scottinet — APPROVED** [review]

**Pull request merged**